app/models/track_type.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class TrackType < ApplicationRecord
+  belongs_to :program, touch: true
+
+  has_paper_trail meta:   { conference_id: :conference_id },
+                  ignore: %i[updated_at]
+
+  validates :title, presence: true
+  validates :description, presence: true
```

Comment:
I wonder if this is truly necessary.

Reply:
Yes, in certain cases the when the type is straight forward then the title itself will be sufficient.